src/styles/variables.scss

```diff
@@ -0,0 +1,12 @@
+$product-font-color: #060b35;
+$button-color: #00acdc;
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  display: flex;
+}
+
+$product-font-color: #060b35;
+$button-color: #00acdc;
```

Comment:
No need to duplicate variables
```suggestion
```
<img width="418" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/3b1301e7-2c9f-4fbf-9622-4e7657f0712d">